### PR TITLE
#3083 - Label Headings - Supporting Partner- Accessibility Fixes

### PR DESCRIPTION
**Pull request by andrepestana-aot**
- Changed the "Login or Signup" to H2:
image

- Changed "Welcome!" to H1 and the other titles to H2:
image
image
image

- Changed the BodyHeader component to set the title header level. In the case of "Search for applications" it should be H1:
image

- For all the other cases that BodyHeader component is used it was set to use an H2 as it was before:
image

### CODE REVIEW

File: sources/packages/web/src/components/generic/BodyHeader.vue [hunk 1]
```diff
@@ -19,12 +23,18 @@
...
+  components: { DynamicHeader },
   props: {
     title: {
       type: String,
       required: true,
     },
+    titleHeaderLevel: {
+      type: Number,
+      default: 1,
+    },
```
Comment (dheepak-aot) [inline sources/packages/web/src/components/generic/BodyHeader.vue:37]: Considering the primary usage of body header being in the body, can the default be H2?

@andrewsignori-aot @guru-aot @sh16011993 @lewischen-aot @katrina-aot-git ?

File: sources/packages/web/src/components/generic/DynamicHeader.vue [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+export default defineComponent({
+  props: {
+    title: {
+      type: String,
+      required: true,
+    },
+    level: {
+      type: Number,
+      required: true,
+      validator: (value: number) => [1, 2, 3].includes(value),
```
Comment (dheepak-aot) [inline sources/packages/web/src/components/generic/DynamicHeader.vue:16]: 👍

File: sources/packages/web/src/components/generic/BodyHeader.vue (hunk 1 above)
Comment (andrewsignori-aot) [reply sources/packages/web/src/components/generic/BodyHeader.vue:37]: Sounds good to me to have the default as 2.

Comment (katrina-aot-git) [reply sources/packages/web/src/components/generic/BodyHeader.vue:37]: Yeah, if it is the first heading of a section but not the page it should probably be h2.

Comment (andrepestana-aot) [reply sources/packages/web/src/components/generic/BodyHeader.vue]: Done. Thanks!

**Review by dheepak-aot — APPROVED** [review]
Looks good @andrepestana-aot . Thanks for doing the changes.

**Pull request merged**